grpc/interceptor/interceptor.go

```diff
@@ -87,13 +88,16 @@ func fieldsFromContext(ctx context.Context) log.Fields {
 		}
 	}
 
-	md := api.MetadataFromContext(ctx)
+	md, ok := metadata.FromIncomingContext(ctx)
+	if !ok {
```

Comment:
Actually, we only need to check for an IncomingContext. The interceptor package is on the server-side (receiving end) of the RPC, so will only have an incoming context.